src/core/icons/fileIcons.ts

```diff
@@ -2011,6 +2011,11 @@ export const fileIcons: FileIcons = {
     { name: 'replit', fileNames: ['.replit'] },
     { name: 'rescript', fileExtensions: ['res'] },
     { name: 'rescript-interface', fileExtensions: ['resi'] },
+    {
+      name: 'duc',
+      fileNames: ['duc.fbs'],
+      fileExtensions: ['duc'],
```

Comment:
Do you want to support "\*.duc" file names or "\*.duc.json"? If the latter is the case then the file extension should be "duc.json"

Reply:
I want to support *.duc and duc.fbs